src/vm/callingconvention.h

```diff
@@ -1415,8 +1415,11 @@ void ArgIteratorTemplate<ARGITERATOR_BASE>::ComputeReturnFlags()
             }
 #endif
 
+#ifndef UNIX_X86_ABI
             if  (size <= ENREGISTERED_RETURNTYPE_INTEGER_MAXSIZE)
```

Comment:
Could I ask a good way to do that? I don't see a trivial way to find out if this is managed or unmanaged.

There is `m_fUnmanagedCallConv` flag which is used only with `FEATURE_INTERPRETER` on but it is not even defined in class members.